package.json

```diff
@@ -69,7 +67,6 @@
     "gulp-replace": "^1.0.0",
     "gulp-sourcemaps": "^2.6.4",
     "gulp-typescript": "^4.0.2",
-    "image-webpack-loader": "^4.3.1",
     "install": "^0.13.0",
```

Comment:
Remove `install` dependency, since its unused.
Probably was added by mistake in previous iterations.